app/conf/config.go

```diff
@@ -165,3 +180,40 @@ func createSqsSubscription(configSubscription app.EnvSubsciption, topicArn strin
 	newSub.SubscriptionArn = subArn
 	return newSub
 }
+
+func setQueueRedrivePolicy(queues map[string]*app.Queue, q *app.Queue, strRedrivePolicy string) error {
+	// support both int and string maxReceiveCount (Amazon clients use string)
```

Comment:
inspired from [gosqs](https://github.com/p4tin/goaws/blob/master/app/gosqs/queue_attributes.go#L32) but decided to not introduce a dependency from `conf` to `gosqs` package but open for suggestion there.